src/code_080068f8.c

```diff
@@ -19,48 +19,175 @@ static s32 D_03000eac; // unknown type, unknown if exists
 // Graphic Control functions
 
 
-#include "asm/code_080068f8/asm_0800679c.s"
+void func_0800679c(u16 *arg0) {
+    *arg0 -= 1;
+}
 
-#include "asm/code_080068f8/asm_080067a4.s"
+struct unk_struct_08006bb4_init *func_080067a4(u16 arg0, u16 arg1, struct BitmapFontBG *arg2, const char *arg3, s16 arg4, s16 arg5, 
+                    struct struct_080067a4_1 *arg6, struct struct_08005e54 *arg7, s16 arg8, s16 arg9, u32 arg10) {
+    struct unk_struct_08006bb4_init *temp_r7;
+    struct struct_080067a4_1_sub *temp_r4;
+
+    temp_r7 = (struct unk_struct_08006bb4_init *)mem_heap_alloc_id(arg0, sizeof(struct unk_struct_08006bb4_init));

```

Comment:
mem heap functions return void pointers specifically so that casts like this aren't necessary